finetune/config.py

```diff
@@ -239,7 +239,7 @@ def get_default_config():
     settings = Settings(
         # General Settings
         low_memory_mode=False,
-        float_16_predict=False,
+        float_16_predict=True,
```

Comment:
oh we've decided to make this the default?

Reply:
I think let's make it opt in for a bit and then change the default once we've built some trust that it won't ever do us harm?  Have no reason to believe this should ever be problematic though given that the gpu_info check is in place.